crates/conformance-test-utils/src/hive/genesis.rs

```diff
@@ -38,6 +38,15 @@ impl HiveGenesisConfig {
     }
 }
 
+// Define constant addresses for Kakarot contracts
+const KAKAROT_ADDRESSES: &[(&str, &str)] = &[
```

Comment:
Do we need these a constants? Also if we do any reason to have a [(str, str)] instead of a HashMap for example? Or just have 6 separate constants (KAKAROT_ADDRESS, CONTRACT_ACCOUNT_ADDRESS,...).
Last question is are `contract_account` and `externally_owned_account` addresses or class hashes?